src/crimson/os/seastore/btree/btree_range_pin.h

```diff
@@ -229,6 +210,10 @@ class BtreeNodeMapping : public PhysicalNodeMapping<key_t, val_t> {
     ceph_assert(parent);
     return parent->is_valid();
   }
+protected:
+  bool is_parent_pending() const {
```

Comment:
`parent` is already protected, probably no need to introduce this interface ?

Reply:
Ah, yes:-)

Reply:
All done:-)